### ISSUE TITLE
[Bug]: World loading log messages: "enhancedcelestials:harvest_moon_crops" is not a valid item tag!

### ISSUE THREAD

**Issue by Not-February** [CorgiTaco/Enhanced-Celestials#99]
### Mod Version

2.0.9

### Mod Loader

Forge

### Mod Loader Version

39.0.59

### What happened?

Whenever I launch a world with Enhanced Celestials installed, this error shows up in the log several times.

I am unsure if it's actually affecting the harvest moon event or not, but since as far as I know that works off item tags to give you more crop drops, something may not have the desired behavior right now.

Since the moon events aren't regular and it doesn't actually crash the game or cause significant problems to normal gameplay, this might otherwise go unnoticed, thus I am sharing it here.

### latest.log

https://gist.github.com/Febilian/3c9b3159b2167d90806aeb36838a5d53

**Comment by Not-February** [comment]
Reasonably sure at this point that Harvest Moons are nonfunctional and I suspect this tag issue is to blame.

**Comment by kaz880** [comment]
Yeah I've got the same problem as you do. I'm pretty sure as well that the tag is to blame.
Out of curiosity, did you happen to see if it was affecting the blood and blue moon as well?

**Comment by Not-February** [comment]
Blue moon worked as intended when I was using it in this version(giving luck based on whether it's a regular or super blue moon), I personally had blood moons turned off because too many mobs were laggy so I couldn't say if it was working as intended(but it was definitely spawning more mobs).

Only the harvest moon actually uses item tags as far as I know, so it would be the only one affected in this way
